posthog/models/organization.py

```diff
@@ -129,13 +129,16 @@ class OrganizationInvite(UUIDModel):
         related_query_name="organization_invite",
         null=True,
     )
+    emailing_attempt_made: models.BooleanField = models.BooleanField(default=False)
     created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True)
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
 
     def validate(self, *, user: Optional[Any], email: Optional[str] = None) -> None:
         if not email:
             assert user is not None, "Either user or email must be provided!"
             email = user.email
+        if self.is_expired():
+            raise ValueError("Invite has expired.")
```

Comment:
Maybe make it a bit more actionable and friendly? "Your invite has expired. Please ask your admin to generate a new one."